talktorials/11_online_apis/11b_online_docking.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>Maybe add proper docstring description to <code>dogsite_scorer_submit_with_pdbid </code> not only comment on that they only allow PDB codes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>